src/app/payment-hub/request-to-pay/incoming-request-export/incoming-request-export.component.html

```diff
@@ -51,11 +49,10 @@
           name="workflowinstancekey"
           placeholder="comma-separated-list"
           id="workflowinstancekey"
-          (keyup)="arrayConvert($event)"
         />
       </mat-form-field>
       <mat-form-field>
-        <mat-label>START DATE</mat-label>
+        <mat-label>START FROM</mat-label>
```

Comment:
Here, you are implicitly assuming that users would know what "Start From" would mean but they won't. 

START DATE was a more accurate label. Please correct it.

Reply:
Resolved